### PR TITLE
styling the widget

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Combined guide of self-hosted and Okta-hosted sign-in widget styling
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-320488](https://oktainc.atlassian.net/browse/OKTA-320488)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience that can be used to authenticate users on any website. How you customize the widget depends on whether Okta is hosting the widget or you are hosting the Widget.
+
+* Self-hosted Sign-in Widget: After you have [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#getting-started) into your project and configured the authentication scenarios that you want to support, you then want to customize. You can apply customizations to match your branding using CSS and JavaScript. See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget).
+
+* Okta-hosted Sign-In Widget: Okta hosts the sign-in page for your applications that appears when your applications redirect to Okta to sign users in. You can customize the Okta-hosted sign-in page using easy controls or an embedded HTML editor that are provided on the **Custom Signin** tab of the **Customization** page in the Developer Console. See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget).
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:9]:
```suggestion
* Okta-hosted Sign-In Widget: Okta hosts the sign-in page for your applications that appears when your applications redirect to Okta to sign users in. You can customize the Okta-hosted sign-in page using easy controls or an embedded HTML editor that is provided on the **Custom Sign In** tab of **Customization** > **Signin Page** in the Developer Console. See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget).
```

Changed to match text in developer console. It's a shame that Customization > Signin Page is not Customization > Sign In Page.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md [hunk 2]
```diff
@@ -64,7 +70,7 @@ To access the application's client ID (which uniquely identifies the application
 </script>
 ```
 
-Elsewhere in your page, using the method above, you can inspect the client ID and take action. For example, if you had a CSS file on your server that was for a particular client's CSS:
+Elsewhere on your page, using the method above, you can inspect the client ID and take action. For example, if you had a CSS file on your server that was for a particular client's CSS:
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:73]: I like `in` better than `on` because you're writing code "in" the file.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+title: Style the Okta-hosted Sign-In Widget
+---
+
+You can add any HTML, CSS, or JavaScript to the **Customization** page that you want. You can also customize the sign-in page per application and with multiple brands.
+
+> **Note:** Before you can get started customizing the Okta-hosted sign-in page, you must have already customized your [Okta URL domain](/docs/guides/custom-url-domain/).
+
+## Edit the sign-in page
+
+The **Customize Sign-In** page offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, from the Developer Console select **Customization**, and then **Signin Page**.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:11]: The title on this page is "Customize Sign-in Page", so maybe Page should be included in bold?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+### Change the Okta Sign-In Widget version
+
+The Okta-hosted sign-in page uses the [Sign-In Widget](https://github.com/okta/okta-signin-widget) component to interact with the user. Changing the Okta Sign-In Widget version allows you to leverage or omit specific Sign-In Widget capabilities by specifying a particular version. If you aren't familiar with the Okta Sign-In Widget, Okta recommends that you select the highest **Major Version** and the latest **Minor Version** (default). For details about the Okta Sign-In Widget capabilities that are supported by major and minor versions, see the [GitHub releases page](https://github.com/okta/okta-signin-widget/releases).
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:15]: I don't know that this line is needed.

> Changing the Okta Sign-In Widget version allows you to leverage or omit specific Sign-In Widget capabilities by specifying a particular version.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+1. To make changes to the major and minor versions, select **Edit** in the **Okta Sign-In Widget Version** section header.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:17]: I don't see an edit button in my developer org. Is this because you have to configure a custom domain first? If so, we might want to say that.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+1. Make changes directly in the embedded editor.
+2. Click **Preview** to preview your changes before you publish.
+3. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
+4. Click **Save and Publish** when you finish.
+
+> **Note:** See the <GuideLink link="../customization-examples">Customization examples</GuideLink> section for examples that you can alter and use on your hosted sign-in page.
+
+## Bypass the Custom Sign-In Page
+
+Use the `/login/default` backdoor sign-in URL to bypass the custom sign-in page. If, for example, something goes wrong with your customizations and your sign-in page won't load, add `/login/default` to the end of your Okta URL to bring up the default sign-in page and sign in:
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:44]:
```suggestion
Use the `/login/default` backdoor sign-in URL to bypass the custom sign-in page. If, for example, something goes wrong with your customizations and your sign-in page won't load, add `/login/default` to the end of your Okta URL to bring up the default sign-in page and log in:
```

Seems weird to have two `sign-in`s so close to each other.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+The following macros in the HTML contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+
+### <span v-pre>`{{pageTitle}}`</span>
+
+Inserts the page title.
+
+Example:
+   ```html
+   <title>{{pageTitle}}</title>
+   ```
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:61]: Should this code block be left-aligned?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+### <span v-pre>`{{{SignInWidgetResources}}}`</span>
+
+Inserts the JavaScript and CSS files required to use the Okta Sign-In Widget.
+
+### <span v-pre>`{{bgImageUrl}}`</span>
+
+Inserts a URL to the background image configured in your Okta organization. This setting can be changed by selecting **Customization**, and then **Appearance**.
+
+Example:
+```javascript
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:72]:
```suggestion
```html
```

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+         "name":"Demo App",
+          "id":"0oadday782XUlHyp90h3"
+       },
+         "issuer":{
+           "name":"default",
+           "id":"ausblkmh242Vtu5430h2",
+          "uri":"https://${yourOktaDomain}/oauth2/default"
+       }
+    }
+   }
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:111]: Left-align JSON code and re-format code to be indented two-spaces.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+For an OpenID Connect application, the application's client ID is stored in the request context's target for that object:
+
+`OktaUtil.getRequestContext().target.clientId`
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:116]: I'd wrap this with backticks and give it a language. For example:

```js
OktaUtil.getRequestContext().target.clientId
```

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+
+There is also additional information available in the `target`, such as `label`.
+
+> **Note:** The `getRequestContext()` method only returns a value when the Okta-hosted sign-in page is loaded in the context of an application (such as SP-initiated flows in SAML or the `/authorize` route for OpenID Connect). Otherwise, it returns undefined.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:120]:
```suggestion
> **Note:** The `getRequestContext()` method only returns a value when the Okta-hosted sign-in page is loaded in the context of an application (such as SP-initiated flows in SAML or the `/authorize` route for OpenID Connect). Otherwise, it returns `undefined`.
```

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+title: Style the self-hosted Sign-In Widget
+---
+
+This section discusses the customization options that you have when you are self-hosting the Sign-In Widget.
+
+> **Note:** You can try all of these customizations yourself using our [Live Widget](https://developer.okta.com/live-widget/).
+
+## Initial sign-in page
+
+You can modify the look of the initial sign-in page using parameters in the `config` section of the main Widget initialization block.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:11]: Does Widget need to be capitalized here?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+      }
+    ]
+  },
+  ...
+};
+```
+
+## Modify the CSS
+
+In addition to the parameters in the Widget's `config`, you can also modify the CSS or override the default styles with your own.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:84]: Can widget be lowercase?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+
+### Modify the existing theme
+
+If you want to add on top of the Okta theme, just edit [okta-theme.scss](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss) and add any CSS to the bottom of the file.
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:88]: I'm not sure this is correct. I've successfully embedded the SIW w/o this file and it doesn't seem to be referenced in our docs anymore.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+
+### CSS customization examples
+
+This section provides examples that you can use to make your own customizations to the widget.
+
+#### Background
+
+```css
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:96]: CSS code blocks are typically indented 2 spaces, not 4. I'd recommend we indent CSS code blocks with 2 spaces.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+
+#### Video Tutorial
+
+For a more in-depth look at styling the widget, you can watch this video:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Q__ugprsOWo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### Live Widget
+
+You can also play with the SASS in real time with the [live widget](https://developer.okta.com/live-widget).
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:170]: According to the [website](https://sass-lang.com/), Sass should not all be capitalized.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+};
+```
+
+For more information about these configuration options, see the [Okta Sign-In Widget Reference page](https://github.com/okta/okta-signin-widget#okta-sign-in-widget).
+
+## Localization
+
+If you want to display different strings depending on the user's language, you can specify this using the following structure:
+
+```javascript
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:207]:
```suggestion
```json
```

**Review by mraible — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md (hunk 5 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:17]: there's a note at the top of the page for that.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md (hunk 6 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md]: we don't use the term "log in". The InfoDev standard is "sign-in". I'll try and reword it.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md (hunk 12 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:11]: yes - it's a thing with and without Okta Sign-In in front of it.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md (hunk 13 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:84]: no because we are referencing the Okta Sign-In Widget - which we've deemed a "proper noun" AFAIK

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md (hunk 14 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md]: ok, I'll remove it.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md [hunk 18]
```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience that can be used to authenticate users on any website. How you customize the widget depends on whether Okta is hosting the widget or you are hosting the Widget.
```
[the remark this answers is not in the document]
Comment (mraible) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md]: I would suggest changing:

> How you customize the widget depends on whether Okta is hosting the widget or you are hosting the Widget.

To:

> How you customize the widget depends on whether Okta is hosting it or you are embedding it in your app.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+3. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
+4. Click **Save and Publish** when you finish.
+
+> **Note:** See the <GuideLink link="../customization-examples">Customization examples</GuideLink> section for examples that you can alter and use on your hosted sign-in page.
+
+## Bypass the Custom Sign-In Page
+
+Use the `/login/default` backdoor sign-in URL to bypass the custom sign-in page. If, for example, something goes wrong with your customizations and your sign-in page won't load, add `/login/default` to the end of your Okta URL to bring up the default sign-in page and enter your credentials:
+
+`https://${yourOktaDomain}/login/default`
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:46]: Instead of single backticks, can you please change this to have three backticks above and below? I think it looks better.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 20]
```diff
@@ -0,0 +1,127 @@
...
+
+## Request context
+
+By calling the `OktaUtil.getRequestContext()` method, JavaScript code on your sign-in page can inspect the current request and make decisions based on the target application or other details.
+
+Here's what is returned from `getRequestContext()`:
+
+```json
+  {
+  "target":{
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:89]: This still needs reformatting to have proper indentation.

**Review by mraible — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/release-notes/2018-19/index.md [hunk 21]
```diff
@@ -23,11 +23,11 @@ You can customize your Okta org by replacing the Okta domain name with a custom
...
 
 ### Custom Error Page is in Early Access
 
-You can customize the text and the look and feel of error pages using an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/custom-hosted-signin/overview/), this feature offers a fully customized error page. For details, see [Configure a custom error page](/docs/guides/custom-error-pages/overview/).
+You can customize the text and the look and feel of error pages using an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), this feature offers a fully customized error page. For details, see [Configure a custom error page](/docs/guides/custom-error-pages/overview/).
 
 ### Bugs Fixed in 2018.19
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018-19/index.md:33]: This file doesn't seem to belong in this pull request.

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2018-19/index.md:33]: I'm fixing the link

Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2018-19/index.md:33]: Okay, I missed that.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience that can be used to authenticate users on any website. How you customize the Widget depends on whether Okta is hosting it or you are embedding it in your app (self-hosting the Widget).
+
+* Self-hosted Sign-in Widget: After you have [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#getting-started) into your project and configured the authentication scenarios that you want to support, you then want to customize. You can apply customizations to match your branding using CSS and JavaScript. See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget).
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:7]: "you then want to customize"  suggested edit:  "you can can then customize"

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+
+* Okta-hosted Sign-In Widget: Okta hosts the sign-in page for your applications that appears when your applications redirect to Okta to sign users in. You can customize the Okta-hosted sign-in page using easy controls or an embedded HTML editor that is provided. Access the **Custom Sign-In Page** in the Developer Console by selecting **Customizetion** and then **Signin Page**. See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget).
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:9]: "the sign-in page for your applications that appears when your applications redirect to Okta" - would suggest shortening to: "the sign-in page that appears when your applications redirect to Okta"

Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:9]: "See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget)" - Is this url complete? It should be going to `docs/guides/style-the-widget/style-self-hosted/`, right?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md (hunk 22 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:7]: "See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget)." - Is the URL complete?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 24]
```diff
@@ -0,0 +1,237 @@
+---
+title: Style the self-hosted Sign-In Widget
+---
+
+This section discusses the customization options that you have when you are self-hosting the Sign-In Widget.
+
+> **Note:** You can try all of these customizations yourself using our [Live Widget](https://developer.okta.com/live-widget/).
+
+## Initial sign-in page
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:9]: Could this subheading, and the other subheadings on this page change from an h2 to an h3? They seem too big.

Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:9]: Or, maybe this could become an h3, and then the sub-topics--logo, custom buttons, links--could become h4s.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md (hunk 22 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md:7]: holy cow, Moti - thank you. TOTALLY missed that.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md [hunk 25]
```diff
@@ -0,0 +1,237 @@
...
+
+##### Video Tutorial
+
+For a more in-depth look at styling the widget, you can watch this video:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Q__ugprsOWo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+##### Live Widget
+
+You can also play with the sass in real time with the [live widget](https://developer.okta.com/live-widget).
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:166]: I think Sass should be capitalized, e.g., https://en.wikipedia.org/wiki/Sass_(stylesheet_language)

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 26]
```diff
@@ -0,0 +1,130 @@
+---
+title: Style the Okta-hosted Sign-In Widget
+---
+
+You can add any HTML, CSS, or JavaScript to the **Customization** page that you want. You can also customize the sign-in page per application and with multiple brands.
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:5]: I was confused by what the Customization page was, since it hadn't been introduced. Would it work to say instead: "You can add any HTML, CSS, or JavaScript you want to the sign-in page"?

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md [hunk 27]
```diff
@@ -0,0 +1,130 @@
...
+https://${yourOktaDomain}/login/default
+```
+
+This URL only bypasses changes that you have made to the HTML in the HTML editor. It doesn't bypass changes that you made using the controls on the left side of the page.
+
+## Use macros and request context
+
+The following macros in the HTML contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+
+### <span v-pre>`{{pageTitle}}`</span>
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:56]: I think these headings for specific template variables are too big. Maybe they would be better as a items in a bulleted list.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md (hunk 26 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:5]: I think it would be good for this intro paragraph to also mention a couple of other topics covered in this page, like macros. Just to give a sense of what's coming up.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md [hunk 28]
```diff
@@ -1,12 +1,15 @@
 ---
 title: Customization examples
 ---
-Use the following examples to help you customize your sign-in page with your own CSS, scripts, and per-application branding.
+
+Use the following examples to help you customize the sign-in page with your own CSS, scripts, and per-application branding.
 
 ## Add your own stylesheet
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:7]: I think this subheading, and the others in this section, should be smaller, maybe h3's, to make it clear they're sub sections of "Customization examples".

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md [hunk 29]
```diff
@@ -55,7 +61,7 @@ To access the application's client ID (which uniquely identifies the application
 <script>
   function getClientId() {
     if (!OktaUtil) return undefined;
-    
+
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:64]: Can poeple find out the unique IDs of their apps in a particular place in the Apps screen of Admin Console?

**Review by motikagan-okta — APPROVED** [review]
lgtm, but left some suggestions and questions.

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md (hunk 25 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md:166]: good catch!

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md (hunk 27 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md:56]: I'll make the h4s - we (Nate and I) wanted them to be headers so that people could bookmark them if they wanted to

File: packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md (hunk 29 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md:64]: good suggestion. thank you.
